source/features/infinite-scroll.tsx

```diff
@@ -27,18 +27,24 @@ const inView = new IntersectionObserver(([{target, isIntersecting}]) => {
 });
 
 function copyFooter(originalFooter: HTMLElement): void {
-	// Copy the footer links to the sidebar to make them more accessible. Also keep a copy in the footer.
-	const footer = originalFooter.cloneNode(true);
+    // Copy the footer links to the sidebar to make them more accessible. Also keep a copy in the footer.
+    const footer = originalFooter.cloneNode(true);
 
-	for (const child of footer.children) {
-		child.classList.remove('pl-lg-4', 'col-xl-3');
-	}
+    for (const child of footer.children) {
+        child.classList.remove('pl-lg-4', 'col-xl-3');
+    }
+
+    // Check if the footer already exists in the sidebar
+    const sidebarFooter = $('[aria-label^="Explore"]').find('.footer');
 
-	$('[aria-label^="Explore"]')!.append(
-		<div className="footer mt-4 py-4 border-top">
-			{footer}
-		</div>,
-	);
+    // If it doesn't exist, append it
```

Comment:
That’s not the problem. The problem is the footer at the footer. The sidebar footer is fine where it is. 

Judging by the code, “copying it to the footer” is actually intentional and not a bug. Perhaps we should do nothing about this.